cdap-app-fabric/src/main/java/io/cdap/cdap/internal/events/SparkProgramStatusMetricsProvider.java

```diff
@@ -86,6 +87,11 @@ public ExecutionMetrics[] retrieveMetrics(ProgramRunId runId) throws MetricRetri
     long startTime = System.currentTimeMillis();
     String runIdStr = runId.getRun();
     String sparkHistoricBaseURL = cConf.get(Constants.Spark.SPARK_METRICS_PROVIDER_HOST);
+    if (Strings.isNullOrEmpty(sparkHistoricBaseURL)) {
+      emitMetrics(runId, startTime, false);
+      logger.error("Spark metrics host is missing.");
```

Comment:
Should be a warn only, not an error.

Reply:
Can you also fix the `logger` variable to `LOG` as well to be consistent with the rest of the codebase.

Reply:
A better log message would be:

```
LOG.warn("The '{}' configuration is missing, no spark metrics will be provided", Constants.Spark.SPARK_METRICS_PROVIDER_HOST);
```

Reply:
Done